index.md

```diff
@@ -11,26 +11,67 @@ layout: default
 
-## Programming Model
-Beam provides a simple and [elegant programming model](https://cloud.google.com/dataflow/model/programming-model) to express your data processing jobs. Each job is represented by a data processing pipeline that you create by writing a program with Beam. Each pipeline is an independent entity that reads some input data, performs some transforms on that data to gain useful or actionable intelligence about it, and produces some resulting output data. A pipeline’s transform might include filtering, grouping, comparing, or joining data.
+Beam currently supports SDKs in the following languages:
 
-Beam provides several useful abstractions that allow you to think about your data processing pipeline in a simple, logical way. Beam simplifies the mechanics of large-scale parallel data processing, freeing you from the need to manage orchestration details such as partitioning your data and coordinating individual workers.
+<table class="table table-condensed">
+<tr>
+  <th>Language</th>
+  <th>SDK Status</th>
+</tr>
+<tr class="success">
+  <td>Java</td>
+  <td>Active Development</td>
+</tr>
+<tr>
+  <td>Python</td>
+  <td>Coming Soon</td>
+</tr>
+<tr>
+  <td>Other</td>
+  <td>TBD</td>
+</tr>
+</table>
 
-## Key Concepts
-* **Simple data representation.** Beam uses a specialized collection class, called PCollection, to represent your pipeline data. This class can represent data sets of virtually unlimited size, including bounded and unbounded data collections.
-* **Powerful data transforms.** Beam provides several core data transforms that you can apply to your data. These transforms, called PTransforms, are generic frameworks that apply functions that you provide across an entire data set.
-* **I/O APIs for a variety of data formats.** Beam provides APIs that let your pipeline read and write data to and from a variety of formats and storage technologies. Your pipeline can read text files, Avro files, and more.
+## Apache Beam Pipeline Runners
 
-See the [programming model documentation](https://cloud.google.com/dataflow/model/programming-model) to learn more about how Beam implements these concepts.
+The Beam Pipeline Runners translate the data processing workflow you define with your Beam program into the API compatible with the distributed processing back-end of your choice. When you run your Beam program, you'll need to specify the appropriate runner for the back-end service where you want to execute your workflow.
+
+Beam currently supports Runners that work with the following distributed processing back-ends:
+
+<table class="table table-condensed">
+<tr>
+  <th>Runner</th>
+  <th>Status</th>
+</tr>
+<tr class="success">
+  <td>Google Cloud Dataflow</td>
+  <td>Generally Available</td>
```

Comment:
I think someone reading this may assume the runner is GA and ready for primetime use. Should we really say this is also in development? @davorbonaci can correct me if the runner is really GA and ready for everyday use.